src/vm/tests/costs.rs

```diff
@@ -49,11 +49,11 @@ pub fn get_simple_test(function: &NativeFunctions) -> &'static str {
         AsMaxLen => "(as-max-len? list-bar u3)",
         Len => "(len list-bar)",
         ListCons => "(list 1 2 3 4)",
-        FetchEntry => "(map-get? map-foo {a 1})",
-        FetchContractEntry => "(contract-map-get? .contract-other map-foo {a 1})",
-        SetEntry => "(map-set map-foo {a 1} {b 2})",
-        InsertEntry => "(map-insert map-foo {a 2} {b 2})",
-        DeleteEntry => "(map-delete map-foo {a 1})",
+        FetchEntry => "(map-get? map-foo (tuple (a 1)))",
```

Comment:
Why revert this to `(tuple ...)` syntax?

Reply:
Tuple literals here failed unit testing. Figuring out why was not within the scope of the PR but likely resolved with #1337 eliminating implicit tuples.

Reply:
Gotcha -- this should be resolved via #1340, which just merged to `master`

Reply:
Seems to do the trick. File reverted to original tuple literals.